# Tarea Clase 2

Ejercicio 1.75

In [ ]:
#import scipy.constants
import matplotlib.pyplot as plt
import numpy as np
import math
#En este código se calcula la variación del derretimiento de hielo en función del espesor
# Datos
T_1 = 0 #[C]
T_2 = 5 #[C]
d = 0.2
"Propiedades"
h_f_agua = 333.7 #[kJ/kg]
k1 = 83.51

"Análisis"
A = math.pi*d**2
e = [i/10000 for i in range(10,101)]
e = np.array(e)
m_dot_hielo = []
Q_dot_hielo = []
for i in range(10,101):
    m_dot_hielo = k1*A*(T_2-T_1)/(e*h_f_agua*1000)
    

plt.plot(e, m_dot_hielo)
plt.xlabel('$Espesor \quad [m]$')
plt.ylabel('$\dot{m}_{hielo} \quad [kg/s]$')
plt.title('Tasa de derretimiento de hielo')
plt.xticks([0, 0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.007, 0.008, 0.009, 0.01])
plt.show()
plt.savefig('Tasa de derretimiento')

Ejercicio 1-97

![](Captura.png)

Suposiciones:
1. Existen condiciones estables
2. K es constante a lo largo del material
3. $\alpha = \epsilon = 0.8$

$\dot{Q}_{cond}$ está relacionado con $T_{superficial}$ 
<br>
$\dot{q}_{cond}+\alpha\dot{q}_{solar}=\dot{q}_{conv}+\dot{q}_{rad}$ (1)
<br>
$\dot{q}_{cond}=k\frac{T_{out}-T_{in}}{L}$ (2)
<br>
$\dot{q}_{conv}=h(T_{out}-T_{alr})$ (3)
<br>
$\dot{q}_{rad}=\epsilon \sigma (T_{out}^4-T_{alr}^4)$ (4)
<br>
$\dot{q}_{solar}=150 [\frac{W}{m^2}]$

In [4]:
#from scipy.constants import k
epsilon = 0.8
alpha = 0.8
#sigma = scipy.constants.k
sigma = 5.67*10**(-8)
espesor = 0.25 #m
h = 8 #W/m2
T_out = 317
T_in = 300
T_alr = 313
q_dot_rad = epsilon*sigma*(T_out**4-T_alr**4)
q_dot_conv = h*(T_out-T_alr)
q_dot_sol = 150
q_dot_cond = q_dot_rad + q_dot_conv - q_dot_sol*alpha
k_conduccion = -1*(q_dot_cond*espesor)/(T_out-T_in)
k_conduccion

0.9605120838588235

# Respuesta:
El valor de la conductividad térmica del material es de 0.96 W/mK. Este valor se asemeja al valor del $\textbf{concreto}$ (0.81 W/mK)